collectd-etcd/README.md

```diff
@@ -0,0 +1,128 @@
+#![](https://github.com/signalfx/integrations/blob/master/collectd-etcd/img/integrations_etcd.png) etcd
+
+_This directory consolidates all the metadata associated with the etcd plugin for collectd. The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-etcd)_
+
+- [Description](#description)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx etcd plugin. Follow these instructions to install the etcd plugin for collectd.
+
+The [`etcd-collectd`](https://github.com/signalfx/collectd-etcd) plugin collects metrics from etcd members from the three [statistics](https://coreos.com/etcd/docs/latest/v2/api.html#statistics) endpoints and the [metric](https://coreos.com/etcd/docs/latest/v2/metrics.html).
+
+#### FEATURES
+
+##### Built-in dashboards
+
+- **etcd Member**: Provides a high-level overview of metrics for a single etcd member.
+
+  [<img src='./img/etcd_member.png' width=200px>](./img/etcd_member.png)
+
+- **etcd Leader**: Provides metrics from the current leader of the cluster.
+
+  [<img src='./img/etcd_leader.png' width=200px>](./img/etcd_leader.png)  
+
+- **etcd Store**: Provides metrics that are specific to a single cluster.
+
+  [<img src='./img/etcd_store.png' width=200px>](./img/etcd_store.png)
+
+### REQUIREMENTS AND DEPENDENCIES
+
+#### Version information
+
+| Software  | Version        |
+|-----------|----------------|
+| collectd  |  4.9 or later  |
+| python | 2.7 or later |
+| etcd | 2.0.8 or later |
+| Python plugin for collectd | (included with [SignalFx collectd agent](https://github.com/signalfx/integrations/tree/master/collectd)[](sfx_link:sfxcollectd)) |
+
+
+### INSTALLATION
+
+1. Download the [collectd-etcd Python module](https://github.com/signalfx/collectd-etcd).
```

Comment:
You might want to tell the customer where to download or at least where to copy this directory.  Your example config correctly tells them to put it in `/usr/share/collectd...`